packages/components/organisms/f-checkout/src/services/tests/daysInMonth.test.js

```diff
@@ -0,0 +1,61 @@
+import getDaysInMonth from '../daysInMonth';
+
+describe('getDaysInMonth', () => {
+    it.each([
+        [31, 'January', 0],
+        [31, 'March', 2],
+        [30, 'April', 8],
+        [31, 'May', 4],
+        [30, 'June', 5],
+        [31, 'July', 6],
+        [31, 'August', 7],
+        [30, 'September', 8],
+        [31, 'October', 9],
+        [30, 'November', 10],
+        [31, 'December', 11]
+    ])('should return %s when month is %s', (expected, monthName, month) => {
+        // Arrange
+        const date = {
+            month,
+            year: 2020
+        };
+
+        // Act
+        const daysInMonth = getDaysInMonth(date);
+
+        // Assert
+        expect(daysInMonth).toEqual(expected);
+    });
+
+    describe('when month is February', () => {
+        const february = 1;
+
+        it('should return 28 for a non-leap year', () => {
```

Comment:
You could include feb in the test above (if you change the year to a non-leap year, and then have the single special-case test for the leap year?

Just thinking it would make the test file easier to understand as it took me a few minutes to figure out why feb was missing above. Could just be me though!

Reply:
Good point, I've updated it